ext/standard/mail.c

```diff
@@ -281,6 +320,11 @@ PHPAPI int php_mail(char *to, char *subject, char *message, char *headers, char
 		efree(f);
 	}
 
+	if (hdr && php_mail_detect_multiple_crlf(hdr)) {
+		php_error_docref(NULL TSRMLS_CC, E_WARNING, "Multiple or malformed newlines found in additional_header");
+		MAIL_RET(0);
```

Comment:
Supposed we have a message like this:
```
To: you
From: me
Subject: text

Please find my thoughts in the attachment

[[attachment.pdf]]]
```
The text *Please find my thoughts in the attachment* should go into the message parameter and *attachment.pdf* should go into the headers.

http://php.net/manual/en/function.mail.php

```
message
    Message to be sent.

    Each line should be separated with a CRLF (\r\n). Lines should not be larger than 70 characters.
```